Weather/Views/CityNameView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CityNameView.swift
+//  Weather
+//
+//  Created by Alex on 5.08.22.
+//
+
+import SwiftUI
+
+struct CityNameView: View {
+    var city: String
+    var date: String
+    
+    var body: some View {
+        HStack {
```

Comment:
Why do you need to put your `VStack` in `HStack` if you don't put anything else there?

Reply:
Fixed.)